pkg/doc/verifiable/test-suite/verifiable_suite_test.go

```diff
@@ -55,7 +56,7 @@ func main() {
 		return
 	}
 
-	privateKey, publicKey := parseKeys(*jwt)
+	privateKey, publicKey := parseKeysES256K(*jwt) // can be replaced with parseKeysRS256()
```

Comment:
ES256K is now used by default to sign VC JWTs (was RS256).
To be substituted by Ed25519.